scripts/evaluate_predicate_classifiers.py

```diff
@@ -1,19 +1,23 @@
 """Script to evaluate learned predicate classifiers on held-out test cases."""
 
-from typing import List, Set
+from typing import Callable, List, Set
 
 import numpy as np
 
 from predicators.src import utils
 from predicators.src.approaches import create_approach
-from predicators.src.approaches.bilevel_planning_approach import \
-    BilevelPlanningApproach
+from predicators.src.approaches.interactive_learning_approach import \
+    InteractiveLearningApproach
 from predicators.src.envs import BaseEnv, create_new_env
 from predicators.src.settings import CFG
 from predicators.src.structs import Action, Predicate, Task
 
 
-def _main() -> None:
+def main(
+    evaluate_fn: Callable[[BaseEnv, InteractiveLearningApproach],
+                          None]) -> None:
+    """Loads an approach and evaluates the predicates using the given
```

Comment:
I like this refactor! It looks like this function is now general, and doesn't necessarily involve the predicates, so let's make the docstring general.